airbyte-integrations/connectors/source-gitlab/source_gitlab/manifest.yaml

```diff
@@ -0,0 +1,607 @@
+    $ref: "#/definitions/base_projects_child_stream"
+    transformations:
+      - "#/definitions/add_project_id_field"
+      - type: AddFields
+        fields:
+          - type: AddedFieldDefinition
+            path: ["commit_id"]
+            value: "{{ (record.get('commit') or {}).get('id') }}"
+    primary_key: "name"
+    $parameters:
+      path: "projects/{{ stream_slice.id }}/repository/branches"
+
+  releases_stream:
+    name: "releases"
+    $ref: "#/definitions/base_projects_child_stream"
+    primary_key: "name"
+    transformations:
+      - "#/definitions/add_project_id_field"
+      - type: AddFields
+        fields:
+          - type: AddedFieldDefinition
+            path: ["author_id"]
+            value: "{{ (record.get('author') or {}).get('id') }}"
+          - type: AddedFieldDefinition
+            path: ["commit_id"]
+            value: "{{ (record.get('commit') or {}).get('id') }}"
+          - type: AddedFieldDefinition
+            path: ["milestones"]
+            value: |
+              {% set ids = [] %}
+              {% for data in record.get('milestones', []) %}
+              {% set _ = ids.append(data.get('id')) %}
+              {% endfor %}
+              {{ ids }}
+    $parameters:
+      path: "projects/{{ stream_slice.id }}/releases"
+
+  tags_stream:
+    name: "tags"
+    $ref: "#/definitions/base_projects_child_stream"
+    primary_key: "name"
+    transformations:
+      - "#/definitions/add_project_id_field"
+      - type: AddFields
+        fields:
+          - type: AddedFieldDefinition
+            path: ["commit_id"]
+            value: "{{ (record.get('commit') or {}).get('id') }}"
+    $parameters:
+      path: "projects/{{ stream_slice.id }}/repository/tags"
+
+  users_stream:
+    name: "users"
+    $ref: "#/definitions/base_projects_child_stream"
+    primary_key: "name"
+    $parameters:
+      path: "projects/{{ stream_slice.id }}/users"
+
+  deployments_stream:
+    name: "deployments"
+    $ref: "#/definitions/base_projects_child_stream"
+    transformations:
+      - "#/definitions/add_project_id_field"
+      - type: AddFields
+        fields:
+          - type: AddedFieldDefinition
+            path: ["user_username"]
+            value: "{{ (record.get('user') or {}).get('username') }}"
+          - type: AddedFieldDefinition
+            path: ["user_full_name"]
+            value: "{{ (record.get('user') or {}).get('name') }}"
+          - type: AddedFieldDefinition
+            path: ["environment_name"]
+            value: "{{ (record.get('environment') or {}).get('name') }}"
+          - type: AddedFieldDefinition
+            path: ["user_id"]
+            value: "{{ (record.get('user') or {}).get('id') }}"
+          - type: AddedFieldDefinition
+            path: ["environment_id"]
+            value: "{{ (record.get('environment') or {}).get('id') }}"
+    $parameters:
+      path: "projects/{{ stream_slice.id }}/deployments"
+
+  # Incremental streams
+  commits_stream:
```

Comment:
The `statistics=1` param has been removed.
I see that the connector performs requests like `https://gitlab.com/api/v4/projects/25156633/repository/commits?with_stats=true&per_page=50&since=2021-01-01T00%3A00%3A00Z&until=2021-06-30T00%3A00%3A00Z` when running in debug mode. Could you please clarify where do you see this param change?

Reply:
Re-running this with the newest changes, I still see the following two changes (which I think I'm fine with):
* `since` and `until` parameters format have changed from `2021-01-01T00:00:00+00:00` to `2021-01-01T00:00:00Z`. The new format is aligned with the documentation example so I'm fine with it 
* The HTTP requests performed changed from having query param `with_stats=True` to `with_stats=true`. I'm not sure if this has an impact though as records seem identical

Do you see the same thing?